src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let numberOfCalls = 0;
+
+  return (numberOne = 0) => {
+    return (numberTwo = 0) => {
+      numberOfCalls++;
+
+      const numbersSum = numberOne + numberTwo;
+
+      return numberOfCalls > 3 && numberOfCalls % 2 === 0
+        ? `Bzzz... Error!`
+        : numbersSum;
```

Comment:
The condition `numberOfCalls > 3 && numberOfCalls % 2 === 0` is used to determine when to return an error message. Verify that this logic aligns with the task requirements, as it will trigger an error on every even call after the third call.